### ISSUE TITLE
When having multiple calendars with the same name: only the first will be displayed

### ISSUE THREAD

**Issue by realestninja** [SimpleMobileTools/Simple-Calendar#2168]
I had three calendars with the same name (let's say "x"), but only one could be displayed.

All of them showed up in "Settings > Manage synced calendars" and can could be activated, but when looking at the calendar view itself, two of them were missing. The two that were missing were also not showing up as filters in the bottom slider menu.

After I renamed the other two calendars into "x2" and "x3", they could be displayed like any other calendar.

Basically: Calendars with the same name seem to collide.
